sdk/examples/poet_python/sawtooth_poet_engine/engine.py

```diff
@@ -66,7 +66,11 @@ def _initialize_block(self):
         return initialize
 
     def _check_consensus(self, block):
-        return self._oracle.verify_block(block)
+        try:
```

Comment:
This should probably be pushed down around the method that does throw this, and return False in that case

Reply:
Sure, sounds good.